### PR TITLE
Kadiatou bio

### PR DESCRIPTION
**Pull request by Kadiatu0**
https://trello.com/c/pMHgL2mA/4-kadiatou-bio-page

Screen Shot 2022-01-11 at 11 11 18 AM
Screen Shot 2022-01-11 at 11 11 38 AM

### CODE REVIEW

**Comment by Kadiatu0** [comment]
The return button is there to take the user back to the home page.

File: src/styles/_Kad_bio.scss [hunk 1]
```diff
@@ -0,0 +1,377 @@
...
+          opacity: 1;
+        }
+      }
+}
+
+
+
+
+  
+  //h1 {
```
Comment (lauren-avilla) [inline src/styles/_Kad_bio.scss:362]: remove comments in lines 362-368

File: src/wintern-bios/2022/TeamLandingPage2022.jsx [hunk 2]
```diff
@@ -1,12 +1,15 @@
 import React, { useState } from 'react';
+import KadiatouComponent from './KadiatouComponent.jsx'
 
 const TeamLandingPage2022 = () => {
   return (
     <div>
         OMG. IT'S A BLANK WEBSITE!
+        {<KadiatouComponent/>}
```
Comment (lauren-avilla) [inline src/wintern-bios/2022/TeamLandingPage2022.jsx:8]: remove curly braces since components don't need curly braces

File: src/Team.jsx [hunk 3]
```diff
@@ -5,23 +5,23 @@ import TeamLandingPage2021 from './wintern-bios/2021/TeamLandingPage2021.jsx';
 import Header from './Header.jsx';
 import Footer from './Footer.jsx';
 
-function Team() {
+function Team({year}) {
 
-    const [year, setYear] = useState(null);
+    const [selectedYear, setSelectedYear] = useState(year);
```
Comment (lauren-avilla) [inline src/Team.jsx:10]: I was just curious... what exactly are these changes doing?

**Review by lauren-avilla — APPROVED** [review]

**Pull request merged**